### ISSUE TITLE
The app is blocked in my country. Again

### ISSUE THREAD

**Issue by THGUYINTHEPICT** [mukul500/AnimeXStream#211]
I can't watch, search or even see the episodes unless I use a VPN. I'm not 100% sure if it's because it's blocked in my country but the signs point at it being so. I said again because the exact same thing happened last year but was fixed(?) Earlier this year. But now it happened again.

**Comment by THGUYINTHEPICT** [comment]
I mean you don't have to fix it but it would be nice to

**Comment by Thegreatfoxxgoddess** [comment]
gogo anime works in your country?

**Comment by THGUYINTHEPICT** [comment]
Yes it does. Perfectly fine. Well I’ve found a workaround anyway which is to just use a vpn everytime I use the app. A bit annoying but it doesn’t bother me that much

**Comment by gamernoob24** [comment]
did you change your dns 
if not then try changing the dns to 
8.8.8.8
8.8.4.4
